### ISSUE TITLE
Sort the card list by learned/review/new

### ISSUE THREAD

**Issue by biza0re** [helloworld1/AnyMemo#363]
Currently the card list colors cards Green for learned, Yellow for review, and Black for unlearned.  I would like to be able to sort a card list by this color.  Currently cards can be sorted by Ordinal / Question / Answer.

**Comment by helloworld1** [comment]
Yes, that's a good feature to add sorting based on the card learning status and difficulty.
